### ISSUE TITLE
Upgrade to bundler v2

### ISSUE THREAD

**Issue by luisramos0** [openfoodfoundation/openfoodnetwork#5193]
### What we should change and why (this is tech debt)
We should upgrade to latest bundler 2.

I cant see why we cant upgrade after we have moved to ruby 2.3. I may be missing something.

### Impact and timeline
I am not really sure.

**Comment by luisramos0** [comment]
I tried to do this but looks like rails is blocking this,, even in rails 4 branch I see:

```
Bundler could not find compatible versions for gem "bundler":
  In Gemfile:
    rails (~> 4.0.13) was resolved to 4.0.13, which depends on
      bundler (>= 1.3.0, < 2.0)

  Current Bundler version:
    bundler (2.1.4)

```

Looks like we will have to wait for later versions of rails before we do this upgrade to bundler 2.

**Comment by Matt-Yorkley** [comment]
We can probably do this now...
